src/software/ai/hl/stp/tactic/passer/passer_tactic.h

```diff
@@ -17,13 +18,9 @@ class PasserTactic : public Tactic
      * Creates a new PasserTactic
      *
      * @param pass The pass this tactic should try to execute
-     * @param ball The ball that we're trying to pass
-     * @param field The field being played on
-     * @param loop_forever Whether or not this Tactic should never complete. If true, the
      * tactic will be restarted every time it completes
      */
-    explicit PasserTactic(Pass pass, const Ball& ball, const Field& field,
-                          bool loop_forever);
+    explicit PasserTactic(Pass pass);
 
     PasserTactic() = delete;
```

Comment:
I've seen this in a couple tactic files, but not all of them - what's the purpose of the 'delete' keyword?

Reply:
This deletes the default constructor for the class, so user of the PasserTactic can create one without a pass.

`auto pt = PasserTactic()` wouldn't compile, only `auto pt = PasserTactic(Pass(...))` would